ci-operator/step-registry/upi/vsphere/windows/pre/upi-vsphere-windows-pre-commands.sh

```diff
@@ -13,10 +13,22 @@ function createWindowsInstanceFile() {
       # set file name
       windows_instance_file="${SHARED_DIR}/${ADDRESS}_windows_instance.txt"
       # create file with instance information
-      echo "username=${USERNAME}" > "${windows_instance_file}"
```

Comment:
Why this change?

Reply:
The username is already available in `${SHARED_DIR}"/windows_vm_template_username.txt`.

On the other hand, `${ADDRESS}_windows_instance.txt` serves as the raw content of the instance information, allowing it to be re-used and extended. Adding just the `username` to the content of the file takes that flexibility away.

If the intention is to maintain a file with just the username of the Windows instance, you should consider something like: `${ADDRESS}_windows_instance_username.txt`.